test_unstructured/partition/epub/test_epub.py

```diff
@@ -8,77 +8,61 @@
 from unstructured.partition.utils.constants import UNSTRUCTURED_INCLUDE_DEBUG_METADATA
 
 DIRECTORY = pathlib.Path(__file__).parent.resolve()
-expected_sections = {
-    "CHAPTER I THE SUN-SEEKER",
-    "CHAPTER II RINKS AND SKATERS",
-    "CHAPTER III TEES AND CRAMPITS",
-    "CHAPTER IV TOBOGGANING",
-    # not included in expected sections because TOC doesn't perfectly match with EpubHtml items
-    # 'CHAPTER V ICE-HOCKEY',
-    "CHAPTER VI SKI-ING",
-    "CHAPTER VII NOTES ON WINTER RESORTS",
-    "CHAPTER VIII FOR PARENTS AND GUARDIANS",
-    "THE FULL PROJECT GUTENBERG LICENSE",
-}
+EXAMPLE_DOCS_PATH = os.path.join(DIRECTORY, "..", "..", "..", "example-docs")
```

Comment:
seen this and other places assume where the root path is in tests so I think we should create a ticket to create a test level fixture called root_path or example_doc_path. https://unstructured-ai.atlassian.net/browse/CORE-2378

Reply:
Good idea.

Reply:
There's a potential issue with a universal EXAMPLE_DOCS_PATH, though, because unlike the `partition` directory, most test files are nested in an additional directory based on their doc type (ppt and pptx are in their own "pptx" folder for example) while others are not. They used to all be in the same directory like the partition files, but I never looked into why this change occurred.